models/comments.models.js

```diff
@@ -12,9 +12,30 @@ function selectCommentsByArticleId(article_id) {
         WHERE article_id = $1
         ORDER BY created_at DESC;
     `, [article_id])
-    .then(result => {
+    .then((result) => {
         return result.rows; 
     });
 }
 
-module.exports = { selectCommentsByArticleId };
+//task 7
+function insertCommentByArticleId(article_id,username,body){
+    if (isNaN(article_id)) {
+        return Promise.reject({ status: 400, msg: 'Bad Request' });
+      }
+    if (!username || !body) {
```

Comment:
sql should throw an error you an error if the username or body is undefined as this will violate the not null constraint you put when creating your comments table